tools/mo/openvino/tools/mo/back/offline_transformations.py

```diff
@@ -37,10 +37,53 @@ def compress_model(func: object):
     from openvino.offline_transformations_pybind import compress_model_transformation  # pylint: disable=import-error,no-name-in-module
     compress_model_transformation(func)
 
-def apply_offline_transformations(input_model: str, framework: str, transforms: list, compress_fp16=False):
+
+def add_layouts(ov_function, argv: argparse.Namespace):
+    from openvino.runtime import Layout  # pylint: disable=import-error,no-name-in-module
+    from openvino.preprocess import PrePostProcessor  # pylint: disable=no-name-in-module,import-error
+
+    prep = PrePostProcessor(ov_function)
+    layout_values = argv.layout_values
+    if '' in layout_values:
+        if len(ov_function.inputs) > 1:
+            input_names = [list(ov_input.get_tensor().get_names())[0] for ov_input in ov_function.inputs]
+            raise Error('Layout without name can be specified for models with only one input, '
+                        'but provided model has {} inputs: \'{}\'. '
+                        'Please specify explicitly input/output name for --layout option'
+                        .format(len(input_names), input_names))
+        layout_values = {
```

Comment:
I think we need to check that a number of inputs equal to one here because it is allowed to have a net Const->Result